dist/smoothscroll.js

```diff
@@ -1 +1,285 @@
-!function(e,t,n){"use strict";function r(){return e.performance!==n&&e.performance.now!==n?e.performance.now():Date.now()}function o(e){return.5*(1-Math.cos(Math.PI*e))}function a(e){if("object"!=typeof e||e.behavior===n||"auto"===e.behavior||"instant"===e.behavior)return!0;if("smooth"===e.behavior)return!1;throw new TypeError(e.behavior+" is not a valid value for enumeration ScrollBehavior")}function l(e,t,n){e.scrollTop=n,e.scrollLeft=t}function i(e){return e.clientHeight<e.scrollHeight||e.clientWidth<e.scrollWidth?e:e.parentNode.parentNode?i(e.parentNode):void 0}function u(t,n){function a(){var s,p,g,d=r(),v=(d-u)/f;return v=v>1?1:v,s=o(v),p=l+(t-l)*s,g=i+(n-i)*s,m(p,g),p===t&&g===n?(l=i=u=null,void e.cancelAnimationFrame(c)):void(c=e.requestAnimationFrame(a))}var l=e.scrollX||e.pageXOffset,i=e.scrollY||e.pageYOffset,u=r();c&&e.cancelAnimationFrame(c),c=e.requestAnimationFrame(a)}function s(n,a){function i(){var t,a,u,v=r(),h=(v-d)/f;return h=h>1?1:h,t=o(h),a=s+(p-s)*t,u=m+(g-m)*t,l(n,a,u),a===p&&u===g?(s=m=d=null,void e.cancelAnimationFrame(c)):void(c=e.requestAnimationFrame(i))}if(n===t.documentElement||n===t.body)return void u(a.left,a.top);var s=n.scrollLeft,m=n.scrollTop,p=a.left,g=a.top,d=r();c&&e.cancelAnimationFrame(c),c=e.requestAnimationFrame(i)}if(!("scrollBehavior"in t.documentElement.style)){var c,f=768,m=e.scrollTo,p=e.scrollBy,g=e.Element.prototype.scrollIntoView;e.scroll=e.scrollTo=function(){return a(arguments[0])?m.call(e,arguments[0].left||arguments[0],arguments[0].top||arguments[1]):u.call(e,~~arguments[0].left,~~arguments[0].top)},e.scrollBy=function(){if(a(arguments[0]))return p.call(e,arguments[0].left||arguments[0],arguments[0].top||arguments[1]);var t=e.scrollX||e.pageXOffset,n=e.scrollY||e.pageYOffset;return u(~~arguments[0].left+t,~~arguments[0].top+n)},Element.prototype.scrollIntoView=function(){var t,n,r,o,l,u;return a(arguments[0])?g.call(this,arguments[0]||!0):(l=i(this),l&&(u=e.getComputedStyle(l,null),n=parseInt(u.getPropertyValue("padding-left"),10),r=parseInt(u.getPropertyValue("padding-top"),10),t={top:this.offsetTop-2*r,left:this.offsetLeft-2*n},o=s(l,t)),o)}}}(window,document);
```

Comment:
You mean _no minifed_ version, right? I would agree with this since it's a better practice to add an external unminified resource to a project and later minified the bundled.